doc/sphinx/changes.rst

```diff
@@ -765,6 +765,15 @@ A few bug fixes and documentation improvements, in particular:
   fixes `#4741 <https://github.com/coq/coq/issues/4741>`_,
   by Helge Bahmann).
 
+Changes in 8.10.2
+~~~~~~~~~~~~~~~~~
+
+**Notations**
+
+- Fixed #11033: regression in not printing coercions to which is also
+  associated a notation (`#11090 <https://github.com/coq/coq/pull/11090>`_,
+  by Hugo Herbelin).
```

Comment:
Please change to the wording I suggested in https://github.com/coq/coq/pull/11090#discussion_r345763408

Reply:
Fixed in #11199.